src/instrumenter.js

```diff
@@ -92,11 +92,12 @@ class Instrumenter {
    * Record a HIT operation.
    * @param {string} status The status of the hit, either "accepted" or "rejected".
    * @param {string} type   The match type, either "rule", "default", or "none".
+   * @param {string} name   The matching rule string name, or null.
    */
-  countHit(status, type) {
+  countHit(status, type, name) {
     this.statsd.increment(`hit.${status}`);
     this.statsd.increment(`hit.${status}.${type}`);
-    this.hitCounter.labels(status, type).inc();
+    this.hitCounter.labels(status, type, name || '').inc();
```

Comment:
You're handling `name || ''` both here and at the call-site. Doesn't hurt anything, though.

Separately, rename the `name` param here to `label`?

Reply:
> Separately, rename the name param here to label?

 good catch, fixed!